pkg/networkservice/common/heal/client.go

```diff
@@ -163,12 +163,11 @@ func (f *healClient) Request(ctx context.Context, request *networkservice.Networ
 	// Set its connection to the returned connection we received
 	req.Connection = rv
 
-	// TODO handle deadline err
-	deadline, _ := ctx.Deadline()
-	duration := time.Until(deadline)
+	// TODO define proper constant timeout
+	timeout := time.Minute
```

Comment:
Why 1 minute here?

Reply:
I think i should remove constant timeout at all...